tools/testmask/main_test.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"os"
+	"testing"
+)
+
+func TestApplyRules(t *testing.T) {
+	tests := []struct {
+		name               string
+		files              []string
+		expectedPackages   []string
+		expectedAcceptance []string
+	}{
+		{
+			name:               "bundle files",
+			files:              []string{"bundle/config.go", "bundle/deploy/deploy.go"},
+			expectedPackages:   []string{"./bundle", "./bundle/deploy"},
+			expectedAcceptance: []string{"bundle"},
+		},
+		{
+			name:               "skip testdata",
```

Comment:
why is this called "skip testdata"?

also, should have a test where testdata alone is modified?